src/login/tests/LoginPage.test.jsx

```diff
@@ -97,67 +97,78 @@ describe('LoginPage', () => {
     store.dispatch = jest.fn(store.dispatch);
 
-    const loginPage = (mount(reduxWrapper(<IntlLoginPage {...props} />))).find('LoginPage');
-    loginPage.find('button.btn-brand').simulate('click');
+    const { container } = render(reduxWrapper(<IntlLoginPage {...props} />));
+    fireEvent.click(container.querySelector('.btn-brand'));
 
     // Check that loginRequestFailure was dispatched and state is updated
-    expect(loginPage.state('errors')).toEqual(errorState);
+
+    expect(screen.getByText('Enter your username or email')).toBeDefined();
     expect(store.dispatch).toHaveBeenCalledWith(loginRequestFailure({ errorCode: 'invalid-form' }));
   });
 
   it('should match state for invalid email (less than 3 characters), on form submission', () => {
-    const errorState = { emailOrUsername: 'Username or email must have at least 3 characters.', password: '' };
     store.dispatch = jest.fn(store.dispatch);
 
-    const loginPage = (mount(reduxWrapper(<IntlLoginPage {...props} />))).find('LoginPage');
+    const { container } = render(reduxWrapper(<IntlLoginPage {...props} />));
 
-    loginPage.find('input#password').simulate('change', { target: { value: 'test', name: 'password' } });
-    loginPage.find('input#emailOrUsername').simulate('change', { target: { value: 'te', name: 'email' } });
-    loginPage.find('button.btn-brand').simulate('click');
+    fireEvent.change(container.querySelector('#password'), { target: { value: 'test' } });
+    fireEvent.change(container.querySelector('#emailOrUsername'), { target: { value: 'te' } });
 
-    expect(loginPage.state('errors')).toEqual(errorState);
+    fireEvent.click(container.querySelector('.btn-brand'));
+
+    expect(screen.getByText('Username or email must have at least 3 characters.')).toBeDefined();
   });
 
-  it('should reset field related error messages on onFocus event', () => {
-    const errorState = { emailOrUsername: '', password: '' };
+  it('should reset field related error messages on onFocus event', async () => {
     store.dispatch = jest.fn(store.dispatch);
 
-    const loginPage = (mount(reduxWrapper(<IntlLoginPage {...props} />))).find('LoginPage');
-    loginPage.find('button.btn-brand').simulate('click');
+    const { container } = render(reduxWrapper(<IntlLoginPage {...props} />));
+
+    await act(async () => {
+      // clicking submit button with empty fields to make the errors appear
+      fireEvent.click(container.querySelector('.btn-brand'));
+
+      // focusing the fields to verify that the errors are cleared
+      fireEvent.focus(container.querySelector('#password'));
+      fireEvent.focus(container.querySelector('#emailOrUsername'));
+    });
 
-    loginPage.find('input#emailOrUsername').simulate('focus');
-    loginPage.find('input#password').simulate('focus');
-    expect(loginPage.state('errors')).toEqual(errorState);
+    // verifying that the errors are cleared
+    // TODO: add expect for password error
```

Comment:
Why are we not adding password `expect` now?

Reply:
I have removed it there is no need to test it